TYPESCRIPT.md

```diff
@@ -220,12 +238,22 @@ const client = contentful.createClient({
   accessToken: '<content-delivery-token>',
 })
 
-type ProductSkeleton = {
-  fields: { relatedProduct: contentful.EntryFields.Entry },
+type ProductSqueleton = {
   contentTypeId: 'product'
+  fields: {
+    productName: contentful.EntryFieldTypes.Text
+    image: contentful.EntryFieldTypes.AssetLink
+    price: contentful.EntryFieldTypes.Number
+  }
+}
+
+type ReferencedProductSkeleton = {
+  fields: { relatedProduct: contentful.EntryFieldTypes.EntryLink<ProductSqueleton> },
+  contentTypeId: 'referencedProduct'
 }
 type Locales = 'en-US' | 'de-DE'
-const entry = client.withoutLinkResolution.getEntry<ProductSkeleton, Locales>('some-entry-id')
+// Chaining withAllLocales to be able to add the Locale type
```

Comment:
I don’t think we need the comment here as we have examples for `withAllLocales` where this is already explained. If it’s becoming too complicated we can also remove the locales here.